### PR TITLE
Add parameters for each inventoried resource

### PR DESCRIPTION
**Pull request by StrDragos**
[sc-7530]

### CODE REVIEW

**Review by xBATx — APPROVED** [review]
🎉 awesome :)

File: com/coralogix/xdr/v1/inventory/inventory_service.proto [hunk 1]
```diff
@@ -26,6 +26,8 @@ message GetCountsRequest {
 }
 
 message GetCountsResponse{
-  //It returns the counts for total number of tests, saas tests, paas tests, and resources count
-  map<string, google.protobuf.UInt32Value> counts = 1;
-}
```
Comment (GMadorell) [inline com/coralogix/xdr/v1/inventory/inventory_service.proto:31]: should we return the execution id here too?
should we add the execution id also to the request?

also, the request should probably look something like this, for consistency:
```
message GetSecurityTestResultsByPolicyCategoryRequest {
  optional google.protobuf.StringValue execution_id = 1;
  optional com.coralogix.xdr.v1.filter.SecurityTestFilter filter = 2;
}
```

Comment (xBATx) [reply com/coralogix/xdr/v1/inventory/inventory_service.proto:31]: yeah that's a good point. Regarding the `SecurityTestFilter` I think that @StrDragos wanted to isolate that type as based on the name - it's saying tests - but yeah I think it'd good to change it too se we're using the same filter everywhere.

Comment (StrDragos) [reply com/coralogix/xdr/v1/inventory/inventory_service.proto:31]: Makes sense. I will apply the change.

Comment (StrDragos) [reply com/coralogix/xdr/v1/inventory/inventory_service.proto:31]: Done

**Review by GMadorell — APPROVED** [review]

**Pull request merged**